tests/test_rule_in_container.sh

```diff
@@ -64,21 +66,23 @@ _arg_datastream=
 _arg_remediate_using="oscap"
 _arg_logdir=
 _arg_dontclean="off"
+_arg_keep_machine_platform="off"
 _arg_dry_run="off"
 _arg_docker="off"
 
 
 print_help()
 {
 	printf '%s\n' "Test a rule using the container backend."
-	printf 'Usage: %s [-n|--name <arg>] [-s|--scenarios <arg>] [-d|--datastream <arg>] [-r|--remediate-using <REMEDIATION>] [-l|--logdir <arg>] [--(no-)dontclean] [--(no-)dry-run] [--(no-)docker] [-h|--help] <rule-1> [<rule-2>] ... [<rule-n>] ...\n' "$0"
+	printf 'Usage: %s [-n|--name <arg>] [-s|--scenarios <arg>] [-d|--datastream <arg>] [-r|--remediate-using <REMEDIATION>] [-l|--logdir <arg>] [--(no-)dontclean] [--(no-)keep-machine-platform] [--(no-)dry-run] [--(no-)docker] [-h|--help] <rule-1> [<rule-2>] ... [<rule-n>] ...\n' "$0"
 	printf '\t%s\n' "<rule>: The short rule ID. Wildcards are supported."
 	printf '\t%s\n' "-n, --name: Name of the test image (default: 'ssg_test_suite')"
 	printf '\t%s\n' "-s, --scenarios: Regex to reduce selection of tested scenarios (no default)"
 	printf '\t%s\n' "-d, --datastream: Path to the datastream to use in tests. Autodetected by default. (no default)"
 	printf '\t%s\n' "-r, --remediate-using: What to remediate with. Can be one of: 'oscap', 'bash' and 'ansible' (default: 'oscap')"
 	printf '\t%s\n' "-l, --logdir: Directory where logs will be stored (no default)"
-	printf '\t%s\n' "--dontclean, --no-dontclean: Dont remove HTML reports from the log directory. (off by default)"
+	printf '\t%s\n' "--dontclean, --no-dontclean: Don't remove HTML reports from the log directory. (off by default)"
+	printf '\t%s\n' "--keep-machine-platform, --no-keep-machine-platform: Don't remove machine platforms. (off by default)"
```

Comment:
This can confuse someone. Does "off by default" mean that if people don't use this option the "machine" platforms will be kept or does it mean that they will be removed?

I suggest renaming this to `--remove-machine-platform` and the default behavior should be to not remove the machine platform.

Reply:
Another naming proposal would be to rename it to `--remove-machine-only` which would be consistent with Automatus as this script is a wrapper over Automatus.